cdisc_rules_engine/dataset_builders/base_dataset_builder.py

```diff
@@ -45,9 +45,10 @@ def __init__(
         self.standard = standard
         self.standard_version = standard_version
         self.library_metadata = library_metadata
+        self.dataset_class = self.data_service.dataset_class
```

Comment:
`dataset_class` is a poor name for this since it conflicts with standards classes. Maybe a better name would be `dataset_implementation`?